lib/Analyses/DynamicAliasAnalysis.cpp

```diff
@@ -167,6 +167,10 @@ void DynamicAliasAnalysis::removePointedBy(Definition Ptr, Location Loc) {
   auto J = PointedBy.find(Loc);
   assert(J != PointedBy.end());
   J->second.erase(Ptr);
+  /*Do not keep those Location entry which does not map to any Defintion set*/
```

Comment:
Thanks for the reply. Should I create another pull request abiding with your suggestion or you will be fixing this on your own?